### ISSUE TITLE
Zakładanie gildi nie działa.

### ISSUE THREAD

**Issue by PrzemoVi** [FunnyGuilds/FunnyGuilds#912]
Siemka.

W spigot 1.13 jest błąd i metoda player.getInventory().containsAtLeast(item,amount) nie działa.
Przez to plugin nie rozpoznaje itemów na gildie w eq i nie da się założyć gildi.

Proponuje zamienić to na coś takiego:

int amount = 0;
for (ItemStack is : p.getInventory().getContents()) {
	if(is != null && is.getType() != null) {
		if (is.getType() == itemToRemove.getType()) {
			amount = amount + is.getAmount();
		}
	}
}

if (amount >= items[i].getAmount()) {
    //kod
}

Gwarantuje to kompatybilność z innymi silnikami bukkit/spigot/paperspigot/cokolwiek.
Przynajmniej do czasu, aż ekipa spigot to naprawi.

Przez ten błąd nie działa mi kilka pluginów i musiałem je przerobić, aby używały tej metody ( w tym starszej wersji fg ), po przeróbce wszystko działa.
Oczywiście używam najnowszego builda spigot - prosto z buildtools.

Pozdro :)

**Comment by Kamilkime** [comment]
Możesz zawsze zrobić pull request do FG =)

**Comment by Kamilkime** [comment]
Na jakich itemach to testowałeś? Bo u mnie... wszystko działa jak należy...

**Comment by PrzemoVi** [comment]
domyślne itemy, stone, cokolwiek. Nic nie działa.
Spróbuj zainstalować najnowszy spigot. Np. z Yive`s mirror

**Comment by Kamilkime** [comment]
Mam najnowszy, `--rev 1.13`, dopiero co budowałem. Pamiętaj, że domyślne itemy (konkretnie stone) ma nazwę, więc musisz mieć dokładnie taki sam

**Comment by PrzemoVi** [comment]
https://i.imgur.com/sSmowvl.png

**Comment by PrzemoVi** [comment]
Tu masz kopie serwera, na której to testowałem:

http://endimc.pl/kopiaserwera.rar

**Comment by Kamilkime** [comment]
No, ale tak jak mówiłem - w configu masz wymagany stone z nazwą:
![image](https://user-images.githubusercontent.com/10383567/43372946-9153acae-93a6-11e8-9ddf-56fa468c6239.png)

A w eq masz już nienazwany:
![image](https://user-images.githubusercontent.com/10383567/43372952-9b652a1a-93a6-11e8-9f53-fb717d0372e9.png)

I stąd cały problem =)

**Comment by PrzemoVi** [comment]
Po wywaleniu z configa nazwy dalej jest problem :/

**Comment by Kamilkime** [comment]
No, u mnie żadnego problemu z tą metodą nie ma, wszystkie itemy prawidłowo wykrywa
